File: code-examples/python/contents/delete.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

headers = {
    'accept': "application/json",
    'content-type': "application/json",
    'authorization': "YourPublicKey:YourToken"
    }

conn.request("DELETE", "/v0/contents/2", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: code-examples/java/brands/delete.java
HttpResponse<String> response = Unirest.delete("http://api.marketcloud.it/v0/brands/:id")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .header("authorization", "YourPublicKey:YourToken")
  .asString();